### PR TITLE
Portfolio add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by raihan2bd**
# I and My coding partner have : 
- I have coded the Html contact form as a driver.
- My partner coded the CSS contact form as a driver.
- In the end, I have fixed the linter error as a driver.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -268,6 +268,41 @@ <h3 class="sub_title">
...
         </ul>
       </section>
+      <section class="contact_form">
+        <h2 class="title">Contact me</h2>
+        <p class="contact_form_des">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form class="form_container">
```
Comment (leonardodiasb) [inline index.html:278]: - [ ] Your form looks good but it is not working properly. To work the form needs the action and method attribute to integrate with [formspree service](https://formspree.io/html). Kindly read the getting started section and apply the formspree service to your form.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @raihan2bd, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Professional PR message. ✔️ 
 - Good commit history. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (raihan2bd) [reply index.html:278]: @leonardodiasb, Thank you so much for your review. I will fix it quickly. Thanks once again fo remained me. 💯  I was forgot to grab the form data. 😊

**Review by christianbiring1 — APPROVED** [review]
Hi @raihan2bd , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# APPROVED :tada: 
imageswelljob

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] The formspree service is used correctly :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**